man/ngircd.8.tmpl

```diff
@@ -53,6 +53,9 @@ Don't fork a child and don't detach from controlling terminal.
 All log messages go to the console and you can use CTRL-C to
 terminate the server.
 .TP
+\fB\-y\fR, \fB\-\-syslog\fR
```

Comment:
Probably let's move this down _below_ the `-V` option, like you did for the `--help` output? Then it is (more or less) sorted alphabetically which I think would be handy ;-) – What do you think?
(And yes, I broke this rule years ago with having `-h` at the end … waaaah!)